charts/rancher-vsphere-cpi/templates/configmap.yaml

```diff
@@ -19,13 +19,11 @@ data:
     vcenter:
       {{ .host | quote }}:
         server: {{ .host | quote }}
-        user: {{ .username }}
-        password: {{ .password }}
         datacenters:
           - {{ .datacenters | quote }}
-    {{- if .labels.generate }}
-    labels:
-      region: {{ .labels.region }}
-      zone: {{ .labels.zone }}
-    {{- end }}
+      {{- if .labels.generate }}
+      labels:
```

Comment:
In the example at https://github.com/kubernetes/cloud-provider-vsphere/blob/master/releases/README.md I see `labels` at the same indentation as `vcenter`.
Is the example wrong and we do in fact need to increase indentation here?

Reply:
I don't believe so but I think it's confusing to have labels as a separate section since labels is actually a subfield of vCenter. In this case, should we have parity with upstream?